test/helpers.ts

```diff
@@ -25,12 +25,20 @@ export function finish(done: (e?: any) => {}) {
   return [() => done(), (e: any) => { done(e); }];
 }
 
-export function runApply(dbAlias: string, done: (e?: any) => {}) {
-  iasql.apply(dbAlias, false, 'not-needed').then(...finish(done));
+export function runApply(dbAlias: string) {
+  return runQuery(dbAlias, 'call iasql_apply();');
```

Comment:
what do you mean by be careful? increase the timeout on the SP? it is 25m right now, should it be more? technically the stored procedure call times out, but the `apply` continues. it would go on to the next part of the test though

Reply:
exactly, if we are doing a count check after the apply it will fail if it have not finished for example. I think time are long enough but well just a heads up to add a comment or something if at some point we have a long running test

Reply:
yes, I think 30m or 45m is ok